app/views/layouts/_flash.slim

```diff
@@ -0,0 +1,18 @@
+- flash.each do |key, value|
+  ruby:
+    puts "key", key, "value", value
```

Comment:
@aleguacaran what's this for?

Reply:
That partial displays flash messages in the layout using Bootstrap alerts